src/main/scala/com/softwaremill/realworld/common/CustomAuthInputs.scala

```diff
@@ -0,0 +1,54 @@
+package com.softwaremill.realworld.common
+
+import sttp.model.HeaderNames
+import sttp.model.headers.WWWAuthenticateChallenge
+import sttp.tapir.{Codec, CodecFormat, DecodeResult, EndpointInput, Mapping, Schema, header}
+
+/** This authorization mechanism grants the ability to employ either the "Bearer" header or the "Token" parameter at the commencement of the
```

Comment:
1. both bearer/token are header values - now it seems one is a header, the other some kind of parameter. More precisely, they are header value prefixes
2. I'd also mention why we want to support both - so that you can authenticate both from swagger & the realworld spec